app/Actions/Discord/Channels/CreateThreadFromRepository.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace App\Actions\Discord\Channels;
+
+use Exception;
+use App\Models\Channel;
+use App\Models\Repository;
+use Illuminate\Support\Str;
+use App\DTO\Discord\ChannelData;
+use App\Enums\Discord\ChannelType;
+use Illuminate\Database\Eloquent\Model;
+
+class CreateThreadFromRepository
+{
+    public function execute(Repository $repository): Model|bool
+    {
+        try {
+            /** @var ChannelData $threadData */
+            $threadData = app(CreateChannel::class)->execute(
```

Comment:
is it ok to create a forum on discord per repository?!

Reply:
Yes ! but we have to re work this pr, lot of change since and i want to make a lot of new thing.

My idea was : One Forum per repo and one thread into the forum per pr